### PR TITLE
DEV-85 Adds get all applicants route with pagination

### PR DESCRIPTION
**Pull request by gunnmgirl**
**What kind of change does this PR introduce? (Bugfix, feature, docs update, ...)**
Feature
**What is the current behavior? (You can also link to an open issue here)**
https://developish.atlassian.net/browse/DEV-85?atlOrigin=eyJpIjoiYWVjYmM5ZjcyYWNlNGJkMWI0N2YyYjNjNDk3OGYyYTgiLCJwIjoiaiJ9
**What is the new behavior (if this is a feature change)?**
Added get all applicants route with pagination
**Does this PR introduce a breaking change?**
No
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] You’ve rebased your work on the current state of the master development tree
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Your changes follow to the coding standards
- [ ] Your code is documented

**Other Information**:

### CODE REVIEW

File: src/applicants/applicantsController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Applicant from "./applicantsModel";
+
+const LIMIT = 8;
+
+const getAllApplicants = async (req, res, next) => {
+  const { page } = req.params;
+  try {
+    const applicants = await Applicant.findAll({
+      limit: LIMIT,
+      offset: Number(page) * LIMIT,
```
Comment (nikolamajmunovic) [inline src/applicants/applicantsController.js:10]: pls use parseInt instead, example `parseInt(some_number, 10)`

File: src/applicants/applicantsController.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Applicant from "./applicantsModel";
+
+const LIMIT = 8;
```
Comment (nikolamajmunovic) [inline src/applicants/applicantsController.js:3]: maybe to save limit on constants? just a suggestion

File: src/applicants/applicantsController.js (hunk 1 above)
Comment (gunnmgirl) [reply src/applicants/applicantsController.js:10]: fixed

File: src/applicants/applicantsController.js (hunk 2 above)
Comment (gunnmgirl) [reply src/applicants/applicantsController.js:3]: fixed

**Review by nikolamajmunovic — APPROVED** [review]

**Pull request merged**